src/ring_theory/ideal_operations.lean

```diff
@@ -538,6 +538,26 @@ end map_and_comap
 
 end ideal
 
+namespace is_ring_hom
+
+variables {R : Type u} {S : Type v} [comm_ring R] [comm_ring S]
+variables (f : R → S) [is_ring_hom f]
+
+def ker : ideal R := ideal.comap f ⊥
+
+@[simp] lemma ker_eq : ((ker f) : set R) = is_add_group_hom.ker f := rfl
```

Comment:
I'm not sure if this should be a simp-lemma.

Reply:
I agree, I was just using `simp` at first to solve these lemmas but in general we don't want it to unfold. I'll change it now.